src/Core/src/Core/IContextFlyoutContainer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Represents a view that can contain a context flyout menu, which is usually represented as a right-click menu.
+	/// </summary>
+	public interface IContextFlyoutContainer
```

Comment:
Waffling a little bit on the name here. 
This would be our first use of suffixing `Container` to mean this. 

We have an [`IContainer` interface](https://github.com/dotnet/maui/blob/main/src/Core/src/Core/IContainer.cs) which might make this a little confusing. 

I feel like `IContextFlyoutElement` is a bit more consistent with our other names. Thought I get how that's a little less direct.

https://github.com/dotnet/maui/blob/main/src/Core/src/Core/IContentView.cs#L8
https://github.com/dotnet/maui/blob/main/src/Core/src/Core/IGraphicsView.cs
https://github.com/dotnet/maui/blob/main/src/Core/src/Core/IMenuBarElement.cs

I do get how `Container` is a bit more descriptive I just worry about pivoting naming.

Reply:
This is probably fine 

We're going to run into this with `Tooltip` as well and should probably look at renaming `IMenuBarElement` and `IToolbarElement`

The other variation here would be something like `IHasContextFlyout`

Reply:
`IContextFlyoutElement` seems fine to me. `IHasXyz` is funny but a little bit "out there" 😄